api/src/identity-access-management/scripts/anonymize-users-in-batch.script.js

```diff
@@ -29,35 +32,53 @@ export class AnonymizeUsersInBatchScript extends Script {
           describe: 'The user id of the anonymizer',
           demandOption: true,
         },
+        chunkSize: {
+          type: 'number',
+          describe: 'Chunk size to process the file',
+          default: CHUNK_SIZE,
+        },
+        delay: {
```

Comment:
`delayInMilliseconds` ?

```suggestion
        delayInMilliseconds: {
```